app/views/documents/form_partials/_validate_document.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with model: [@planning_application, @document], local: true, class: "document", builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
```

Comment:
This is creating a form nested within another form, which is incorrect. You should remove this `form_with`